### PR TITLE
Release 2.3.1

### PR DESCRIPTION
**Pull request by jonasscheid**
Small release:
- Merge template update 2.13.1
- Remove `Channel.view` statement that has accidently sneaked its way into the previous release
- Add check for protein map to prevent failure if no information is available (@christopher-mohr)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.3.1 - Oesterberg Hotfix - 2024-03-25
+
+### `Fixed`
+
+- [#237](https://github.com/nf-core/epitopeprediction/pull/237) - Update to nf-core template `2.13.1`
```
[the remark this answers is not in the document]
Comment (jonasscheid) [reply CHANGELOG.md:10]: Not really, see PR description

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.3.1 - Oesterberg Hotfix - 2024-03-25
```
[the remark this answers is not in the document]
Comment (jonasscheid) [reply CHANGELOG.md:6]: Agreed 😁

**Comment by christopher-mohr** [comment]
> Have to check what's going on with biomart again...

It just seems to randomly fail within CI (now again). Last time I could not reproduce it on GitPod...

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.3.1 - Bühl - 2024-03-28
```
Comment (christopher-mohr) [inline CHANGELOG.md:6]:
```suggestion
## v2.3.1 - Oesterberg- 2024-03-28
```

Still Oesterberg... ;)

**Comment by christopher-mohr** [comment]
@jonasscheid What's the status here?

**Comment by jonasscheid** [comment]
I thought we are currently stuck because of some random ci tests failing. I'll do the template merge this week and look at again

**Comment by christopher-mohr** [comment]
> I thought we are currently stuck because of some random ci tests failing. I'll do the template merge this week and look at again

I think this was just the BioMart connection issue that happens from time to time. The test profile also runs fine on GitPod. I can do the template merge later today or tomorrow morning then we see if the problem still exists (on CI).

**Comment by jonasscheid** [comment]
Great, thank you! 🙏

**Review by christopher-mohr — APPROVED** [review]
LGTM

File: workflows/epitopeprediction.nf [hunk 4]
```diff
@@ -415,22 +415,44 @@ workflow EPITOPEPREDICTION {
...
+
+    ch_multiqc_files = ch_multiqc_files.mix(
+        ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
+    ch_multiqc_files = ch_multiqc_files.mix(ch_collated_versions)
+    ch_multiqc_files = ch_multiqc_files.mix(
+        ch_methods_description.collectFile(
+            name: 'methods_description_mqc.yaml',
+            sort: true
+        )
+    )
```
Comment (mapo9) [inline workflows/epitopeprediction.nf:455]: is this the way to go now?
I find this very hard to read...

**Review by mapo9 — APPROVED** [review]
lgtm

File: workflows/epitopeprediction.nf (hunk 4 above)
Comment (jonasscheid) [reply workflows/epitopeprediction.nf:455]: Agreed! I guess the excessive indenting forces prettier to linebreak...

**Pull request merged**